packages/framework/src/Resources/views/Admin/Form/numberSlider.html.twig

```diff
@@ -0,0 +1,13 @@
+{% block number_slider_widget %}
```

Comment:
nitpick, in this file, your tabs have 8 spaces, there should be just 4 spaces for the tab